crates/loro-internal/src/diff_calc.rs

```diff
@@ -262,24 +262,15 @@ impl DiffCalculator {
                             InternalContainerDiff {
                                 idx,
                                 bring_back,
-                                is_container_deleted: are_rest_containers_deleted,
+                                is_container_deleted: false,
```

Comment:
Don’t we need to determine this anymore?

Reply:
It seems like we don't need this field

Reply:
I'll test it in another branch.